sections/03_categorize.md

```diff
@@ -47,53 +47,67 @@ regards to data availability and labeling.
-evaluations. This theme of an ensemble  between deep learning algorithm and
-human expert may help overcome some of the  challenges presented by data
-limitations.
+with deep learning approaches. Ciresan et al. [@tag:Ciresan2013_mitosis]
+developed one of the earliest examples, winning the 2012 International
+Conference on Pattern Recognition's Contest on Mitosis Detection while
+achieving human competitive accuracy. Their approach uses what has become a
+standard convolutional neural network architecture trained on public data.
+In more recent work, Wang et al. [@tag:Wang2016_breast_cancer] analyzed stained
+slides to identify cancers within slides of lymph node slices. The approach
+provided a probability map for each slide. On this task a pathologist has about
+a 3% error rate. The pathologist did not produce any false positives, but did
+have a number of false negatives. Their algorithm had about twice the error
+rate of a pathologist. However, their algorithms errors were not strongly
+correlated with the pathologist. Theoretically, combining both could reduce
+the error rate to under 1%. In this area, these algorithms may be ready to
+incorporate into existing tools to aid pathologists. The authors' work suggests
+that this could reduce the false negative rate of such evaluations. This theme
+of an ensemble between deep learning algorithm and human expert may help
+overcome some of the challenges presented by data limitations.
 
 One source of training examples with rich clinical annotations is the electronic
-health record. Recently Lee et al.[@doi:10.1101/094276] developed an approach to
-distinguish individuals with Age-related Macular Degeneration from control
-individuals. They extracted approximately 100,000 images from structured
-electronic health records, which they used to train and evaluate a deep neural
-network. Combining this data resource with standard deep learning techniques,
-the authors reach greater than 93% accuracy. One item that is important to note
-with regards to this work is that the authors used their test set for evaluating
-when training had concluded. In other domains, this has resulted in a minimal
-change in the estimated accuracy
-[@url:http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf].
+health record. Recently Lee et al.[@tag:Lee2016_emr_oct_amd] developed an
+approach to distinguish individuals with Age-related Macular Degeneration
+from control individuals. They extracted approximately 100,000 images from
+structured electronic health records, which they used to train and evaluate a
+deep neural network. Combining this data resource with standard deep learning
+techniques, the authors reach greater than 93% accuracy. One item that is
+important to note with regards to this work is that the authors used their test
+set for evaluating when training had concluded. In other domains, this has
+resulted in a minimal change in the estimated accuracy
```

Comment:
It was not my contribution and it was approved before, so I don't feel convinced to remove it right away. Maybe it's worth looking at once again at the final paper reviewing stage?